doc/usermanual/darkroom/modules/tone/filmicrgb.xml

```diff
@@ -0,0 +1,619 @@
+                (e.g. clouds).
+              </para>
+
+              <para>
+                The module is derived from the same named module in Blender 3D modeller by T. J.
+                Sobotka. While it is primarily intended to recover high-dynamic range from raw
+                sensor data, it can be used with any image in replacement of the base curve
+                module. The developer provided a detailed explanation of the module in a video
+                called <ulink url="https://www.youtube.com/watch?v=zbPj_TqTF88">Filmic RGB:
+                remap any dynamic range in darktable 3.0</ulink>.
+              </para>
+
+              <para>
+                Filmic rgb is the successor of the filmic module provided in Darkroom 2.6. While
+                the underlying principles did not change much, users of the previous version
+                should not expect a 1:1 translation of their workflow, and may find
+                <xref linkend="filmic-rgb-for-26-users"/> useful.
+              </para>
+            </entry>
+            <entry>
+              <graphic fileref="darkroom/modules/images/filmic.png" scalefit="1" width="80%"
+                align="center"/>
+            </entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+  </sect4>
+
+  <sect4>
+
+    <title>Prerequisites</title>
+
+<!-- IHU - my understanding is that the prerequisites don't change much, except for the 
+      exposure settings - which I modified. Is that correct? -->
+
+    <para>
+      In order to get the best out of filmic rgb, images need some preparation:
+      <itemizedlist>
+
+        <listitem><para>
+          In-camera, expose the shot <quote>to the right</quote>. This implies under exposing
+          the shot so that the highlights are at the right of the histogram, just on the verge
+          of clipping, but not clipped. It does not matter if the picture preview is very dark
+          on your camera screen: as long as highlights are unclipped, filmic rgb should be able
+          to recover details from the raw data. Beware that clipped data are not recoverable.
+          Some cameras have a clipping alert preview to help you diagnose this, and some even
+          have an highlight-priority exposure mode.
+        </para></listitem>
+
+        <listitem><para>
+          In the <link linkend="exposure"><emphasis>exposure</emphasis></link> module, push the
+          exposure until the midtones are clear enough. Do not worry about losing the
+          highlights: they will be recovered as part of the filmic rgb processing. However, it
+          is important to avoid negative pixels in black areas: the computations done by filmic
+          rgb will result in unpredictable results in this case. For some cameras models (Canon,
+          mainly), rawspeed, the raw decoding library of darktable, may set an exaggerated black
```

Comment:
My suggestion here is for an extra set of parentheses, so that the text "the raw decoding library of darktable" doesn't look like the second item in a list. Feel free to disregard, if you don't agree. :)

For some cameras models (Canon, mainly), rawspeed (the raw decoding library of darktable), may set an exaggerated....